apps/src/lib/kits/maker/ui/SetupGuide.jsx

```diff
@@ -158,6 +184,73 @@ const downloadButtonStyle = {
   textAlign: 'center'
 };
 
+class MicrobitDescription extends React.Component {
+  render() {
+    return (
+      <div>
+        <h2>The micro:bit</h2>
+        <center>
+          <a href={'https://microbit.org/'}>
+            <img
+              src={'maker/microbit-drawing-green.png'}
+              width={200}
+              style={style.microbitImg}
+            />
+          </a>
+        </center>
+        <SafeMarkdown
+          markdown={
+            'The [micro:bit](https://microbit.org/) is a circuit board designed by the BBC with a variety of sensors and other components, like a compass and an 8 by 8 array of programmable LEDs.'
```

Comment:
instead of having multiple `SafeMarkdown` components to get the break in content we want, we should be able to use markdown to do that. the translatable string would contain the newline(s) and we would only need 1 instance of `SafeMarkdown`

Reply:
Yeah, I was actually wondering the same. 
The only reason I did separate safeMarkdowns because I saw that being done in the same file:
 ```
       <h2>{i18n.support()}</h2>
        <SafeMarkdown markdown={i18n.debugMakerToolkit()} />
        <SafeMarkdown markdown={i18n.contactGeneralSupport()} />
```
I can thing about two considerations:
1. Would longer/more complex strings make possible to introduce translation errors that could potentially break the page? This strings seem to be pretty simple without many escaped characters. 
2. Can the new html structure be different from the original? A different html layout may result in failing eyes test. Separate safeMarkdowns create a div for each line, this is different from the original structure.

Reply:
both good questions!
1. possibly! i'm not sure how much additional risk this change would add, but the Foundations team might feel differently since they're the experts in this area.
2. yes, the HTML structure can be different as long as the visual layout doesn't change too much (unless product wants to change the visuals, of course). it sounds "bad" when eyes tests fail, but making changes like this will always cause some change, which is fine (and good)